plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaSecurityConfig.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.kafka.security.KafkaEndpointIdentificationAlgorithm;
+import io.trino.plugin.kafka.security.KafkaKeystoreTruststoreType;
+import io.trino.plugin.kafka.security.KafkaSecurityConfig;
+import io.trino.testing.assertions.Assert;
+import org.testng.annotations.Test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
+import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
+import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestKafkaSecurityConfig
+{
+    @Test
+    public void testDefaults()
+    {
+        // No defaults, should be empty if nothing is set specifically
+        assertRecordedDefaults(recordDefaults(KafkaSecurityConfig.class)
+                .setKeystoreLocation(null)
+                .setKeystorePassword(null)
+                .setKeystoreType(null)
+                .setKeyPassword(null)
+                .setTruststoreLocation(null)
+                .setTruststorePassword(null)
+                .setTruststoreType(null)
+                .setEndpointIdentificationAlgorithm(null));
+    }
+
+    @Test
+    public void testExplicitPropertyMappings()
+            throws IOException
+    {
+        String secret = "confluent";
+        Path keystorePath = Files.createTempFile("keystore", ".jks");
+        Path truststorePath = Files.createTempFile("truststore", ".jks");
```

Comment:
Do you really need existing files here? I am not sure if this test is able to validate that. If so can you please create negative tests that configuration fails as expected for invalid configuration when given files does not exists. (Then tests in `TestKafkaPlugin` for that might not be needed).